### PR TITLE
add skedge hostname to resolve the url

### PR DESCRIPTION
**Pull request by mohitsuman**
Add skedge related hostname to the hydra environment.

### CODE REVIEW

**Comment by mohitsuman** [comment]
@gandhikeyur Please have a look.

**Comment by gandhikeyur** [comment]
@engineersamuel Could you please review and merge.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hydrajs",
-  "version": "0.0.15",
+  "version": "0.0.16",
```
Comment (engineersamuel) [inline package.json]: Change this back to 0.0.15 then I'll merge

**Comment by mohitsuman** [comment]
@engineersamuel I have rolled back the version. Please review and merge.

**Pull request merged**